app/models/user.js

```diff
@@ -21,5 +24,9 @@ export default Addressable.extend({
 
   fullName: Ember.computed('firstName', 'lastName', function(){
     return (this.get('firstName') + " " + this.get('lastName'));
+  }),
+
+  isInfoComplete: Ember.computed('firstName', 'lastName', 'email', 'mobile', function(){
+    return this.get('firstName') && this.get('firstName').length !== 0 && this.get('lastName') && this.get('lastName').length !== 0 && this.get('email') && this.get('email').length !== 0 && this.get('mobile') && this.get('mobile').length !== 0 && this.get('title') && this.get('title').length !== 0;
```

Comment:
No need to compare length with 0. Also why in model and not in controller?

Reply:
I thought  model state information should be within model itself. Had tried with this.get('firstName') only but it didn't work. So , had to explicitly add length check.